src/scss/partials/_header.scss

```diff
@@ -6,6 +6,10 @@ $header-separator                   : 1px;
 $header-separator                   : 4px;
 $header-border-color                : $grey--lightest;
 $header-buttonIcon-color            : $blue;
+$header-buttonCount-bg              : $blue;
```

Comment:
Can you add in the alternative Menulog theme colour for this (jusr set it to green in the theme  overrides below)